<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>valgrind::MemcheckConfig</class>
 <widget class="QWidget" name="valgrind::MemcheckConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>475</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Memcheck</string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Extra &amp;Parameters:</string>
        </property>
        <property name="buddy">
         <cstring>memcheckParameters</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="memcheckParameters"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Freelist size</string>
        </property>
        <property name="buddy">
         <cstring>freeListSize</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KIntNumInput" name="freeListSize" native="true">
        <property name="value" stdset="0">
         <number>10000000</number>
        </property>
        <property name="minimum" stdset="0">
         <number>0</number>
        </property>
        <property name="relativeValue" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="suffix" stdset="0">
         <string> bytes</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="showReachable">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial,Geneva,Helvetica,sans-serif'; font-size:medium; color:#202020;&quot;&gt;When disabled, the memory leak detector only shows &amp;quot;definitely lost&amp;quot; and &amp;quot;possibly lost&amp;quot; blocks.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial,Geneva,Helvetica,sans-serif'; font-size:medium; color:#202020;&quot;&gt;When enabled, the leak detector also shows &amp;quot;reachable&amp;quot; and &amp;quot;indirectly lost&amp;quot; blocks.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Show reachable</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="undefValueErrors">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Controls whether Memcheck reports uses of undefined value errors.</string>
        </property>
        <property name="text">
         <string>Undef Value Errors</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
